api/event.js

```diff
@@ -0,0 +1,33 @@
+const dbAccess = require("../db");
+const express = require("express");
+const router = express.Router();
+/**
+ * GET Route that handles single event attendance
+ * @param eventName takes in an event name
+ * @returns json object with number of attendances and number of staff members
+ */
+router.get("/eventAttend/:eventName", async (req, res) => {
+  let attendanceCount = 0;
+  let staffMembers = 0;
+  try {
+    dbAccess.db.query("SELECT * FROM attendance", (error, response) => {
```

Comment:
Hi! SQL's a pretty powerful query language and you can really simplify this route by using a `WHERE` clause (some documentation [here](https://www.w3schools.com/sql/sql_where.asp)) and a `COUNT`. Otherwise, it gets pretty inefficient to select the entire table as it grows to potentially thousands of rows, transfer it over the network, and then loop over everything each time this route is called.

Reply:
Hi! Thanks for the tip. We will be changing this route this week so @justinyaodu could you implement the query with the `WHERE`clause?